restli-int-test-api/src/main/idl/com.linkedin.restli.examples.greetings.client.autoValidationDemos.restspec.json

```diff
@@ -1,7 +1,7 @@
 {
   "annotations" : {
     "createOnly" : {
-      "value" : [ "stringB", "intB", "UnionFieldWithInlineRecord/com.linkedin.restli.examples.greetings.api.myRecord/foo2", "MapWithTyperefs/*/id" ]
+      "value" : [ "stringB", "intB", "UnionFieldWithInlineRecord/com.linkedin.restli.examples.greetings.api.myRecord/foo2", "MapWithTyperefs/*/id", "ArrayWithInlineRecord/*/bar3" ]
```

Comment:
What is the wildcard meaning in here

Reply:
This is a pathspec indicating which fields are `CreateOnly`. In this schema, the wildcard indicates that it matches any element of the array field `ArrayWithInlineRecord`. Thus, `ArrayWithInlineRecord/*/bar3` means that the `bar3` field of any element in the `ArrayWithInlineRecord` array field is `CreateOnly`.